libs/Database.js

```diff
@@ -382,15 +392,18 @@ class Database {
    * @param {Array} indexes array of string containing the name of the indexes to create
```

Comment:
should add a doc string here for the new params parameter

Reply:
done